mon-pix/tests/integration/components/user-logged-menu-test.js

```diff
@@ -147,7 +147,7 @@ describe('Integration | Component | user logged menu', function() {
         });
       });
 
-      it('should render a button link to the profile when the user is not on compte page', async function() {
+      it('should render a button link to the profile when the user is not on profile page', async function() {
```

Comment:
`expect(find('.user-menu-item__details-account-link').textContent.trim()).to.equal('Mon compte');` => maintenant le bouton ne devrait pas être 'Mon profil' ?

Reply:
Le test a été supprimé entre-temps dans la PR d'ajout du menu de navigation.

Reply:
C'est pas juste le test, c'est aussi l'implémentation. Est-ce que le bouton `Mon compte` existe toujours ou pas ?